src/server.py

```diff
@@ -78,6 +81,37 @@ def get_date_range(dates):
                 'endDate': max(dates).strftime('%Y-%m-%d %H:%M')}
 
 
+def valve(head_rev, pushlog_id):
+    """Determine which kind of job should been returned"""
+    low_job_return = True
+    COUNTER.increase_the_counter()
+    request_list = []
+    try:
+        request_list = session.query(TaskRequests.head_rev, TaskRequests.pushlog_id,
+                                     TaskRequests.low_job_return).all()
```

Comment:
in this case, we have a head_rev of a push- we have <500/day, and we only care about a small range.

One thing to keep in mind- we need to track this per branch, right now: mozilla-inbound, fx-team, and autoland.  There is a possibility we need to do other branches if we extend SETA past failure discovery.  Ideally we would have this for each job to support more priorities- but I think we can make good assumptions for branch/revision.